web/src/components/tutorials.astro

```diff
@@ -142,328 +144,328 @@ const tutorials = [
     description: "Example to demonstrate the federated site policies for authorization, resource and data privacy management.",
-    link: "https://github.com/NVIDIA/NVFlare/blob/main/examples/advanced/federated-policies/README.rst"
+    link: `https://github.com/NVIDIA/NVFlare/blob/${gh_branch}/examples/advanced/federated-policies/README.rst`
   },
   {
     title: "Custom Authentication",
     tags: ["adv.", "deployment"],
     description: "Example to demonstrate the custom authentication policy and secure mode.",
-    link: "https://github.com/NVIDIA/NVFlare/tree/main/examples/advanced/custom_authentication/README.rst"
+    link: `https://github.com/NVIDIA/NVFlare/tree/${gh_branch}/examples/advanced/custom_authentication/README.rst`
   },
   {
     title: "Job-Level Authorization",
     tags: ["adv.", "deployment"],
     description: "Example to demonstrate the job-level authorization policy and secure mode.",
-    link: "https://github.com/NVIDIA/NVFlare/tree/main/examples/advanced/job-level-authorization/README.md"
+    link: `https://github.com/NVIDIA/NVFlare/tree/${gh_branch}/examples/advanced/job-level-authorization/README.md`
   },
   {
     title: "Federated Learning Hub",
     tags: ["adv.", "deployment"],
     description: "Allow hierarchical interaction between several levels of NVIDIA FLARE FL systems, e.g. Tier-1 (hub) and Tier-2 (sub-systems).",
-    link: "https://github.com/NVIDIA/NVFlare/tree/main/examples/advanced/fl_hub/README.md"
+    link: `https://github.com/NVIDIA/NVFlare/tree/${gh_branch}/examples/advanced/fl_hub/README.md`
   },
   {
     title: "Getting Started",
     tags: ["beg.", "algorithm", "client-api", "model-controller", "job-api", "pytorch", "lightning", "sklearn", "tensorflow"],
     description: "Getting started examples using the Client API, Model Controller API, and Job API for different frameworks.",
-    link: "https://github.com/NVIDIA/NVFlare/tree/main/examples/getting_started"
+    link: `https://github.com/NVIDIA/NVFlare/tree/${gh_branch}/examples/getting_started`
   },
   {
     title: "Step-by-Step CIFAR10 Examples",
     tags: ["beg.", "algorithm", "pytorch", "dl"],
     description: "Step-by-step examples series with CIFAR-10 (image data) to showcase different NVIDIA FLARE features, workflows, and APIs.",
-    link: "https://github.com/NVIDIA/NVFlare/tree/main/examples/hello-world/step-by-step/cifar10"
+    link: `https://github.com/NVIDIA/NVFlare/tree/${gh_branch}/examples/hello-world/step-by-step/cifar10`
   },
   {
     title: "Step-by-Step HIGGS Examples",
     tags: ["beg.", "algorithm", "pandas", "sklearn", "xgboost"],
     description: "Step-by-step examples series with HIGGS (tabular data) to showcase different NVIDIA FLARE features, workflows, and APIs.",
-    link: "https://github.com/NVIDIA/NVFlare/tree/main/examples/hello-world/step-by-step/higgs"
+    link: `https://github.com/NVIDIA/NVFlare/tree/${gh_branch}/examples/hello-world/step-by-step/higgs`
```

Comment:
Can this "example link" also be substituted with a variable? But we could do that in another PR if we want

Reply:
which link are you referring to? These github links I've replaced the branch with a substitution